src/sagemaker/session.py

```diff
@@ -3222,7 +3222,9 @@ def create_model_package_from_containers(
         )
 
         def submit(request):
-            if model_package_group_name is not None:
+            if model_package_group_name is not None and not model_package_group_name.startswith(
+                "arn:"
```

Comment:
is the startswith`arn` check enough? to cover all such cases?

Reply:
yes, this check would be sufficient. confirmed with madbal@